### ISSUE TITLE
Set meta data for a PDF

### ISSUE THREAD

**Issue by todda00** [Gottox/node-pdfutils#22]
I have the package setup to read PDF keywords and title correctly, is there currently any way to write new information (keywords, title, etc.) to a PDF and save the file?

**Comment by Gottox** [comment]
No. pdfutils has very limited support for writing PDFs. This may change in future, but at the moment we are bound to the features poppler and cairo are providing.
